### ISSUE TITLE
Replies from fediverse are returned as if from instance

### ISSUE THREAD

**Issue by mediaformat** [jdormit/pterotype#8]
Here is the scenario:
- WP Pterotype post from @blog@ptero.urbanwildlife.biz.
- A masto user @django@social.coop replies.
- comment is approved.
- WP sends same approved comment to Masto as @django@ptero.urbanwildlife.biz

**Comment by jdormit** [comment]
So just to be clear, are you seeing duplicate comments? E.g. a Mastodon user posts a reply, which becomes a WordPress comment, which is then posted back to Mastodon as a separate Mastodon reply?

**Comment by mediaformat** [comment]
Yes, exactly!

See if you can replicate it, by replying to the « new test » toot from @blog@ptero.urbanwildlife.biz

**Comment by mediaformat** [comment]
![4429a688-a75e-42c0-9d31-b44f6a6b1491](https://user-images.githubusercontent.com/2895351/48414356-77351900-e718-11e8-865f-3d3df39eb745.png)

**Comment by jdormit** [comment]
Hey @mediaformat, I left a reply on that post. Once you approve it, let's see if it gets duplicated.

**Comment by mediaformat** [comment]
Hey, didn't see the reply until just now.

So, there was just your comment there to approve. 
The minute I approved it though, a second comment was created using your username@ptero.urbaniwildlife.biz as the associated email.

Also of interest, the WP created comments, although remaining unapproved in WP, are visible on Masto.
![ptero-comments-2](https://user-images.githubusercontent.com/2895351/48721092-3002c880-ebef-11e8-8404-d83bd4691234.jpeg)

**Comment by mediaformat** [comment]
I've tried modifying Discussion settings of wordpress.
I had the following 2 settings enabled:
-  Attempt to notify any blogs linked to from the article 
-  Allow link notifications from other blogs (pingbacks and trackbacks) on new articles 

Tooted some more replies after disabling them, but it hasn't changed the behaviour.

Also, I've left a Toot on qa.getpterotype.com, might help debug.

**Comment by jdormit** [comment]
Yup, it definitely got duplicated. Can you confirm the version of Pterotype you are running?

**Comment by mediaformat** [comment]
Version 1.2.11
